js/validation.js

```diff
@@ -0,0 +1,62 @@
+
+<<<<<<< HEAD
+const validateTypeOfKey = function (key, value, type) {
+  if (typeof value !== type) {
+    throw new Error(`${key} has to be ${type}`)
+=======
```

Comment:
This looks like merge conflict markers from Github committed in the source code 😨